crates/volta-layout/src/v3.rs

```diff
@@ -19,11 +19,13 @@ layout! {
             "inventory": inventory_dir {
                 "node": node_inventory_dir {}
                 "npm": npm_inventory_dir {}
+                "pnpm": pnpm_inventory_dir {}
```

Comment:
I'm not 100% sure, it may be okay that this is added to the existing layout, rather than included in a migration. We'll need to be sure that it's backwards-compatible, otherwise we'll have to create a migration (which is one of the more awkward areas of the code base). So I'm hopeful that it works and doesn't have issues 🤞🏻

Reply:
This is implemented via https://github.com/volta-cli/volta/pull/1273/commits/1ef45ec9e09b67354d015a2cc2d92f3143a811cd#diff-19d1763ea1b8954788fa4de2fd0d62975d8334c65ca927ddad712d33ef738831R168 thanks to it we don't need to update to layout.v4 to achieve the goal.

Reply:
Makes sense, thanks!